Basic_SmartConfig/app/application.cpp

```diff
@@ -36,5 +8,5 @@ void init()
 
 	WifiAccessPoint.enable(false);
 	WifiStation.enable(true);
-	smartconfig_start(SC_TYPE_ESPTOUCH, smartconfigDone);
+	WifiStation.smartConfigStart(SCT_EspTouch);
```

Comment:
WifiStation.smartConfigStart must have a callback as parameter. A Sming-style smartConfigDelegate should be fine.

Since smartconfig works only when AP is off and STA is on then may be the following two commands can be inside of the body of WifiStation.smartConfigStart

```C
  WifiAccessPoint.enable(false);
  WifiStation.enable(true);	
```